test/Client/ClientFactoryTest.cs

```diff
@@ -0,0 +1,68 @@
+using GitHub.Octokit.Client;
+using GitHub.Octokit.Client.Middleware;
+using Xunit;
+
+
+public class TestHandler1 : DelegatingHandler { }
+public class TestHandler2 : DelegatingHandler { }
+
+public class ClientFactoryTests
+{
+
+    [Fact]
+    public void Creates_Client_With_Default_Timeout()
+    {
+        var clientFactory = ClientFactory.Create();
+        Assert.Equal(TimeSpan.FromSeconds(100), clientFactory.Timeout);
+    }
+
+    [Fact]
+    public void Creates_Client_Persists_Set_Timeout()
+    {
+        var clientFactory = ClientFactory.Create();
+        clientFactory.Timeout = TimeSpan.FromSeconds(5);
+        Assert.Equal(TimeSpan.FromSeconds(5), clientFactory.Timeout);
+    }
+
+    [Fact]
+    public void Create_Returns_NonNullHttpClient()
+    {
+        HttpMessageHandler handler = new HttpClientHandler();
+        var client = ClientFactory.Create(handler);
+        Assert.NotNull(client);
+    }
+
+    [Fact]
+    public void CreateDefaultHandlers_Returns_Expected_Handlers()
+    {
+        var handlers = ClientFactory.CreateDefaultHandlers();
+        Assert.Contains(handlers, h => h is APIVersionHandler);
+        Assert.Contains(handlers, h => h is UserAgentHandler);
+    }
+
+    // ChainHandlersCollectionAndGetFirstLink
```

Comment:
What's the purpose of this comment and the below one on line 58? Are they going to be expanded upon?